Examples/Detectors/DD4hepDetector/CMakeLists.txt

```
@@ -20,9 +20,11 @@ add_library(
   src/DD4hepGeometryService.cpp
   src/DetUtils.cpp)
 
-file(GLOB _odd "${CMAKE_SOURCE_DIR}/thirdparty/OpenDataDetector/factory/*")
+file(GLOB _odd "${CMAKE_SOURCE_DIR}/thirdparty/OpenDataDetector/factory/tracker/*")
 if(NOT _odd)
     message(WARNING "The ODD directory seems not to be present. Did you run init the submodule?")
+else()
+  add_compile_definitions(ODD_ACTS_EXTENSION)
```

```suggestion
  target_compile_definitions(ActsExamplesDetectorDD4hep PRIVATE ODD_ACTS_EXTENSION)
```

I think this would be better as it only adds the definition to the one target rather than everything after this. I think PRIVATE should be enough here.